shell/platform/darwin/macos/framework/Source/FlutterViewController.mm

```diff
@@ -438,6 +438,10 @@ - (void)viewWillAppear {
   [self listenForMetaModifiedKeyUpEvents];
 }
 
+- (void)viewDidAppear {
+  [_engine updateWindowMetricsForViewController:self];
+}
```

Comment:
This seems very reasonable, but I'm curious why you added this. Did something not work? Could we add a test for this scenario?

Reply:
I thought I removed it. I don't think this is needed. I will test removing it.